base/sparse/abstractsparse.jl

```diff
@@ -2,7 +2,7 @@
 
 abstract type AbstractSparseArray{Tv,Ti,N} <: AbstractArray{Tv,N} end
 
-const AbstractSparseVector{Tv,Ti} = AbstractSparseArray{Tv,Ti,1}
+const AbstractSparseVector{Tv,Ti} = Union{AbstractSparseArray{Tv,Ti,1}, Base.ReinterpretArray{Tv,1,T,<:AbstractSparseArray{T,Ti,1}} where T}
```

Comment:
ewww

Reply:
also, this was true with the old definition of `reinterpret` on SparseArrays, it's not true with the new definition (see comment on nonzeroinds)